README.md

```diff
@@ -3,18 +3,27 @@
 The Crossmint SDK allows you to offer wallet-less credit card purchases on your NFT drop. It takes 10 lines of code and 5 min to integrate, and is free to use for the seller. You’ll get the sales proceeds in SOL/ETH as if the user was paying with their own wallet.
 
 Supported chains:
-* Solana
-* Polygon (private beta, contact us at sales at crossmint.io)
-* Ethereum L1 (March 2021)
+
+-   Solana
+-   Polygon (private beta, contact us at sales at crossmint.io)
+-   Ethereum L1 (end of Feb 2021)
 
 To get started:
+
 1. Fill out our form in https://www.crossmint.io/developers/onboarding (we'll review your project in under 24hr!)
 2. Follow the instructions below to integrate with your code
 
 ## Demo of the user experience:
 
+# REPLACE
```

Comment:
I have added this as a reminder to myself to replace the old gif with a new one. Will do that shortly.

Reply:
Actually, maybe its better to take care of it later and have a little discussion on what to show in the gif, (or if we even want a gif there)